viral/wheel_freeze_movement.py

```
@@ -0,0 +1,291 @@
+#     previous_rotary_end += position[-1]
+def plot_motion_energy_results(mouse_name: str, date: str) -> None:
+
+    motion_energy_pre = np.load(
+        TEMP_CACHE_PATH / "motion_energy" / f"{mouse_name}_{date}_pre.npy"
+    )
+    motion_energy_pre = motion_energy_pre[1:]
+    events_pre = process_motion_energy(motion_energy_pre, plot=True)
+
+    session_path = CACHE_PATH / f"{mouse_name}_{date}.json"
+    session = Cached2pSession.model_validate_json(session_path.read_text())
+    1 / 0
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)